src/lib/services/documents-service.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * @module services for interacting with CRM v2 documents
+ */
+const documentConnector = require('../connectors/crm-v2/documents');
+const Document = require('../../lib/models/document');
+const errors = require('../errors');
+const documentMapper = require('../mappers/document');
+
+/**
+ * Get CRM documents for a given licence number
+ * @param {String} licenceNumber
+ * @return {Promise<Array>}
+ */
+const getDocuments = async licenceNumber => {
+  const data = await documentConnector.getDocuments(licenceNumber);
```

Comment:
I think this is OK because for the the list API views, I expect them to respond with an empty array and 200 OK, whereas if requesting a specific resource that doesn't exist, I expect them to respond with 404 not found